x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -463,28 +465,62 @@ class AnnotationsTableUI extends Component {
         // find the original annotation because the table might not show everything
         const annotationId = annotation._id;
         const originalAnnotation = annotations.find((d) => d._id === annotationId);
-        const editAnnotationsTooltipText = (
+        const editAnnotationsText = (
           <FormattedMessage
             id="xpack.ml.annotationsTable.editAnnotationsTooltip"
             defaultMessage="Edit annotation"
           />
         );
-        const editAnnotationsTooltipAriaLabelText = i18n.translate(
+        const editAnnotationsAriaLabelText = i18n.translate(
           'xpack.ml.annotationsTable.editAnnotationsTooltipAriaLabel',
           { defaultMessage: 'Edit annotation' }
         );
         return (
-          <EuiToolTip position="bottom" content={editAnnotationsTooltipText}>
-            <EuiButtonIcon
-              onClick={() => annotationUpdatesService.setValue(originalAnnotation ?? annotation)}
-              iconType="pencil"
-              aria-label={editAnnotationsTooltipAriaLabelText}
-            />
-          </EuiToolTip>
+          <EuiButtonEmpty
+            size="xs"
+            aria-label={editAnnotationsAriaLabelText}
+            iconType="pencil"
+            onClick={() => annotationUpdatesService.setValue(originalAnnotation ?? annotation)}
+          >
+            {editAnnotationsText}
+          </EuiButtonEmpty>
         );
       },
     });
 
+    if (this.state.jobId && this.props.jobs[0].analysis_config.bucket_span) {
```

Comment:
Seems like we keep using `this.props.jobs[0]` throughout the file. Can we make a constant to access this job?